cypress/e2e/1-getting-started/todo.cy.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
+
+// Welcome to Cypress!
+//
+// This spec file contains a variety of sample tests
+// for a todo list app that are designed to demonstrate
+// the power of writing tests in Cypress.
+//
+// To learn more about how Cypress works and
+// what makes it such an awesome testing tool,
+// please read our getting started guide:
+// https://on.cypress.io/introduction-to-cypress
+
+describe("example to-do app", () => {
```

Comment:
You have these in `gitignore` but they are still being tracked. Try deleting them from your project, committing that delete then adding them again

Reply:
Done